packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@endpass/ui",
-  "version": "0.19.3",
+  "version": "0.19.5",
```

Comment:
it should be 0.19.4. increment for one point only

Reply:
it should not, because 0.19.4 is in pr of vault
I just prevent version conflict

Reply:
then, in near pr of ui, you should update version to 0.19.6.
in npm you did not have 0.19.4 and please use increase numbering for this.
just fix to `0.19.4`